### ISSUE TITLE
[Nested admins] List of items is not filtered by parent

### ISSUE THREAD

**Issue by jgrygierek** [sonata-project/SonataAdminBundle#4915]
#### Sonata packages

```
sonata-project/admin-bundle              3.x-dev 629f0c4
sonata-project/block-bundle              3.11.0        
sonata-project/cache                     2.0.1           
sonata-project/core-bundle               3.9.0      
sonata-project/datagrid-bundle           2.3.1     
```
#### PHP version

```
7.1.9
```

## Subject

I use nested admins feature. In version 3.31 of sonata admin it is ok. But when I upgrade bundle to dev version, it is not working anymore.

## Expected results

List of items should be filtered by parent id.

## Actual results

List of items contents all existing items from db table.

**Comment by jordisala1991** [comment]
Can you check with this patch applied? #4913

**Comment by jgrygierek** [comment]
Yes, it works.

By the way, title of this PR #4913 is not very user-friendly... Usually when I'm looking for something, I look at titles, not content of PR or issue...

**Comment by jordisala1991** [comment]
Fixed in #4913
